src/Math-Core/PMVector.class.st

```diff
@@ -308,6 +308,13 @@ PMVector >> sqrt [
 	1 to: self size do: [ :n | self at: n put: (self at: n) sqrt ]
 ]
 
+{ #category : #arithmetic }
```

Comment:
@hernanmd I don't recall learning about the idea described in the comment in Linear Algebra (Pure Mathematics). This could be my ignorance so could you point me to a reference?

If this isn't a Pure Mathematics idea, and given it looks like one from Statistics, should we categorize it as such? Open question